README.md

```diff
@@ -99,6 +99,12 @@ To get started:
 
 It will open demo app with component previews on `localhost:4000`. You can change components and they will be updated on page reload. Component previews located in `demo/test/components/previews`.
 
+To run tests:
+
+```bash
+bin/rails test
```

Comment:
True, `rake` wasn't working for me initially so switched to `bin/rails test`.

But looking at it now both commands have the same issue for me, I have to manually add the gems `pry` and `webdrivers` to the Gemfile to be able to run tests. Otherwise require complains about the gem not being present.

Reply:
Ok, I will take a look at require issue.

Reply:
Thanks!